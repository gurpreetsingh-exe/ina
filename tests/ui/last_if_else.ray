// COMPILE_FAIL

fn main() {
    let a = if true {
        if true { 50 } else { 20. }
    } else {
        20.
    };
}

