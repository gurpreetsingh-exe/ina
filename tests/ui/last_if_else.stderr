error: `if` and `else` have incompatible types
   --> last_if_else.ina:5:9
    |
 LL |         if true { 50 } else { 20. }
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `i64`, found `f32`

error: `if` and `else` have incompatible types
   --> last_if_else.ina:4:13
    |
 LL |     let a = if true {
 ...| 
 LL |     };
    | 

